modules/nodes-clusters-cgroups-2.adoc

```diff
@@ -0,0 +1,192 @@
+// Module included in the following assemblies:
+//
+// * nodes/clusters/nodes-cluster-cgroups-2.adoc
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+ifeval::["{context}" == "nodes-cluster-cgroups-2"]
+:nodes:
+endif::[]
+ifeval::["{context}" == "post-install-cluster-tasks"]
+:post:
+endif::[]
+
+:_content-type: PROCEDURE
+[id="nodes-clusters-cgroups-2_{context}"]
+= Configuring Linux cgroup v2
+
+ifdef::post[]
+You can enable link:https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html[Linux control group version 2] (cgroup v2) on specific nodes in your cluster by using a machine config. The {product-title} process for enabling cgroup v2 disables all cgroups version 1 controllers and hierarchies.
+
+[IMPORTANT]
+====
+{product-title} cgroups version 2 support is a Technology Preview feature only. Technology Preview features
+are not supported with Red Hat production service level agreements (SLAs) and
+might not be functionally complete. Red Hat does not recommend using them
+in production. These features provide early access to upcoming product
+features, enabling customers to test functionality and provide feedback during
+the development process.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+endif::post[]
```

Comment:
@peer-review-squad I could not add the TP snippet inside of an ifdef. I added the text manually.